Cargo.toml

```diff
@@ -51,8 +51,12 @@ itertools = "0.10.5"
 log = "0.4.20"
 simplelog = "0.12.1"
 rustc-hash = "1.1.0"
+indexmap = "2.0.1"
 
 [dev-dependencies]
 httptest = "0.15.4"
 tempdir = "0.3.7"
-maplit = "1.0.2"
\ No newline at end of file
+maplit = "1.0.2"
+zksync-web3-rs = "0.1.1"
+ethers-contract = "2.0.7"
```

Comment:
nit: iirc `ethers_contract` is re-exported by `ethers` at `ethers::contract`- you don't need a separate dependency for it